openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java

```diff
@@ -607,6 +607,50 @@ public Response patchApplication(
     return response;
   }
 
+  @PATCH
+  @Path("/fqn/{fqn}")
```

Comment:
we use /name eveywhere for fqn in other apis, lets keep that naming convention

Reply:
I will update the naming convention to use /name for fqn in the APIs and update the PR.